src/modules/Dashboard/features/RespondentData/RespondentDataReview/RespondentDataReview.types.ts

```diff
@@ -79,6 +81,12 @@ export type SingleMultiSelectPerRowItemAnswer = {
   'data-testid'?: string;
 };
 
+export type SliderRowsItemAnswer = {
```

Comment:
Does it make sense generate new type via generic `CreateItemAnswer` here and above in `...ItemAnswer` types?

Reply:
So there are different types, in the first case - formatted data for the summary tab, and in here - simply decrypted data for the response tab

Reply:
I meant another generic type, different but that looks like `CreateFormattedReposense`

Reply:
done